functions/Get-DbaTraceFlag.ps1

```diff
@@ -0,0 +1,96 @@
+﻿FUNCTION Get-DbaTraceFlag {
+	<#
+		.SYNOPSIS
+			Gets SQL Trace Flags information for each instance(s) of SQL Server.
+
+		.DESCRIPTION
+			The Get-DbaTraceFlag returns connected SMO object for SQL Trace Flags information for each instance(s) of SQL Server.
```

Comment:
No need to repeat the command name, can just say command or maybe reword to:

> Returns Trace Flags that are enabled globally on each instance(s) of SQL Server as an object.

Since it does not return an SMO just saying "object" is fine. As well, we need to call out that this only gets global trace flags...I know should be obvious but best to state it.